.classpath

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you delete this file?

Reply:
These files keep coming back whenever I try switching branches. So I just added them to my .gitignore.